package.json

```diff
@@ -130,6 +131,7 @@
     "chai-as-promised": "^7.0.0",
     "chai-files": "^1.2.0",
     "co": "^4.6.0",
+    "dir-compare": "^1.4.0",
```

Comment:
What are we using this for?

Reply:
Comparing the directory structure and files for a Broccoli-builder vs broccoli 2.0 build to ensure they’re identical.

Reply:
Hmm, I think in general I'd prefer to use something like:

```
expect(fixturify.readSync(oneDir)).to.deep.equal(fixturify.readSync(twoDir))
```

Reply:
the more you know. Will test that.

Reply:
Will that compare file contents/hash?

Reply:
@rwjblue are there already tests that already assert the output of the build is correct?